content/en/docs/concepts/configuration/overview.md

```diff
@@ -77,7 +77,7 @@ The [imagePullPolicy](/docs/concepts/containers/images/#updating-images) and the
 
 - `imagePullPolicy: IfNotPresent`: the image is pulled only if it is not already present locally.
 
-- `imagePullPolicy: Always`: the image is pulled every time the pod is started.
+- `imagePullPolicy: Always`: it resolves the image with registry and if it resolves to a sha that is not already pulled to the node, pulls the image. If the resolved sha is already pulled to the node, it uses the already-pulled content.
```

Comment:
First suggestion
```suggestion
- `imagePullPolicy: Always`: it resolves the image with registry and if it resolves to a digest that is not already pulled to the node, pulls the image. If the resolved digest is already pulled to the node, it uses the already-pulled content.
```

Reply:
Second suggestion:
```suggestion
- `imagePullPolicy: Always`: every time the kubelet launches a container, the kubelet queries the container image registry to resolve the name to an image digest. If the kubelet has a container image with that exact digest cached locally, the kubelet uses its cached image; otherwise, the kubelet downloads (pulls) the image with the resolved digest, and uses that image to launch the container.
```

Reply:
+1 to @sftim's change. This helps add a bit more clarity to the definition.

Reply:
@sftim 
Corrected, thank you very much, your changes can really clarify the meaning of imagePullPolicy Always